newIDE/electron-app/app/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Even if it's not a big deal, you should not include this file in your commit - always check/add to your commit only the file that are useful for your PR (this one is not) ;)